### ISSUE TITLE
XMB UI text disappears in Lakka when scrolling through huge collections

### ISSUE THREAD

**Issue by OGWillikers** [libretro/Lakka#485]
I've got a huge collection on a network drive, running on an RPi 3 and screenshot thumbnails enabled.  If I go scrolling through a list of games for about 5-10 minutes, the thumbnail disappears, and a few more scrolls later all text on the screen starts flickering and disappears.  Demanding games seem to perform poorly when this occurs, if I can blindly launch one.

Disabling thumbnails either stops or greatly increases the time to failure (has only happened once on an older build that I've seen) and using boxart thumbnails greatly decreases time.

**Comment by gouchi** [comment]
Thank you for your report. Is it possible to have a video or a picture of the issue ?

**Comment by OGWillikers** [comment]
I'll get on it.  Gotta rescan my collection since I moved some things.

**Comment by Kivutar** [comment]
Looks like the bug found by @lollo78

**Comment by OGWillikers** [comment]
Yeah, that sounds the same.  Apologies for the duplicate.  I can still create a video if it'd be useful.  Looks like the video posted on the other issue is down.

**Comment by Kivutar** [comment]
It's difficult to find the cause of this bug, since it doesn't happen on PC.

**Comment by Kivutar** [comment]
Can you try to increase the memory attributed to the gpu in config.txt?

Try to set `gpu_mem_1024=512`

**Comment by lollo78** [comment]
I'll die before to see this bug fixed. 😂
